packages/form-types-bundle/.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: php
 
 php:
-    - 7.4
+    - 8.1
```

Comment:
I thought we have already removed the Travis checks and replaced them with the Github actions :thinking: :smile:

Reply:
You're right. I removed all Travis configurations and file a new user-story to use GitHub Actions in releaser